### PR TITLE
[enriched/github2] Add 'user_login' field

### PR DESCRIPTION
**Pull request by zhquan**
This code adds the field `user_login` for all enriched items
when the category is `issue` or `pull_request`.

Test updated accordingly.

Signed-off-by: Quan Zhou <quan@bitergia.com>

### CODE REVIEW

File: grimoire_elk/enriched/github2.py [hunk 1]
```diff
@@ -449,7 +455,7 @@ def __get_rich_pull(self, item):
 
         user = pull_request.get('user_data', None)
         if user is not None and user:
-            rich_pr['user_name'] = user['name']
+            rich_pr['user_name'] = user['login']
```
Comment (sduenas) [inline grimoire_elk/enriched/github2.py:458]: Why this change? I see that in this same file there are other references to 'user_name' that are not changed.

Comment (zhquan) [reply grimoire_elk/enriched/github2.py:458]: I changed this by mistake

**Review by sduenas — APPROVED** [review]
LGTM
